VOKUtilities.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "VOKUtilities"
-  s.version          = "0.9.1"
+  s.version          = "0.9.2"
```

Comment:
I think [semver](http://semver.org/) says this should be `0.10.0`, since it adds new features without breaking compatibility.